timereport/chalicelib/lib/add.py

```diff
@@ -10,7 +10,10 @@ def create_event(url, data):
     :return: current_request.raw_body.decode() (same data you send in)
     """
     headers={'Content-Type': 'application/json'}
-    res = requests.post(url=url, data=data, headers=headers)
+    print(f'data is: {data}')
```

Comment:
Use `logging` instead of print

Reply:
well this is just temporary during development.
Logging means we need to do a bunch of extra stuff such as:
`
import logging
logger = logging.getLogger()
dir_path = os.path.dirname(os.path.realpath(__file__))
config = parse_config(f'{dir_path}/../config.json')
logger.setLevel(config['log_level'])

# and then use logging properly... 
`
and since this is temporary we can just remove the print statements after we've done with development. Thoughts?

Reply:
No that is not necessary. You only need to setup the logger and then use correct facility. I will provide my suggestion when I'm not on a mobile

Reply:
sure done!